src/runtime/pay-client.ts

```diff
@@ -416,17 +373,12 @@ export class RedirectVerifierHelper {
     return !!(
       supportsLocalStorage &&
       crypto &&
-      crypto.getRandomValues &&
```

Comment:
is this intended?

Reply:
Yep! TypeScript was complaining that the check was unnecessary